packages/clay-css/src/content/form_custom.html

```diff
@@ -252,6 +252,35 @@ <h3>Custom File</h3>
 	</div>
 </div>
 
+<div class="clay-site-row-spacer row">
+	<div class="col-md-12">
+		<h3>Custom Range</h3>
+
+		<div class="alert alert-warning">
+			This component needs the prefixes: <code>-webkit-appearance: none; -moz-appearance: none;</code>
+		</div>
+
+		<div class="sheet">
+			<label for="customRange">Custom range example</label>
+			<input class="custom-range" id="customRange" type="range" min="0" max="100"/>
```

Comment:
One concern I have is that it may be more complicated to add the tooltip to the thumb when it is moving.

Reply:
Yeah, native tooltips for the `title` attribute usually disappear on interaction and don't show up again until a `mouseleave > mouseenter` sequence. It shouldn't be that hard to track mousemovement and align the tooltip to the it somehow?

Reply:
Maybe not, but we would have to position the tooltip according to the thumb position, I haven't tested it yet but I don't know if we can get the thumb position when moving